executor-scripts/linux/ipv6

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+[ -n "$VERBOSE" ] && set -x
+
+yesno() {
+	case "$1" in
+	yes|YES|true|TRUE|1)
+		echo 1
+		;;
+	*)
+		echo 0
+		;;
+	esac
+}
+
+is_int() {
+	if [ -n "$1" ] && [ "$1" -eq "$1" ] 2>/dev/null; then
```

Comment:
Using this function will result in a subshell due to the process substitution, that's inefficient. The right way to do this is by exploiting the implicit return value instead, like so:

```
is_int() {
	[ -n "$1" ] && [ "$1" -eq "$1" ] 2>/dev/null
}
if is_int 123; then echo yey; fi
if is_int asd; then echo nay; fi 
```

Abusing -eq to check for a number seems a bit of a hack but I can't think of anything much better either.